client/Cargo.toml

```diff
@@ -12,20 +12,20 @@ description = "Embed a substrate node into your subxt application."
 keywords = ["parity", "substrate", "blockchain"]
 
 [dependencies]
-async-std = "1.6.4"
-futures = { version = "0.3.5", features = ["compat"] }
+async-std = "1.8.0"
+futures = { version = "0.3.9", features = ["compat"], package = "futures" }
 futures01 = { package = "futures", version = "0.1.29" }
 jsonrpsee = "0.1.0"
-log = "0.4.11"
-sc-network = { version = "0.8.0", default-features = false }
-sc-service = { version = "0.8.0", default-features = false }
-serde_json = "1.0.57"
-sp-keyring = "2.0.0"
-thiserror = "1.0.20"
+log = "0.4.13"
+sc-network = { version = "0.8.1", default-features = false }
+sc-service = { version = "0.8.1", default-features = false }
+serde_json = "1.0.61"
+sp-keyring = "2.0.1"
+thiserror = "1.0.23"
 
 [dev-dependencies]
-async-std = { version = "1.6.4", features = ["attributes"] }
-env_logger = "0.7.1"
+async-std = { version = "1.8.0", features = ["attributes"] }
```

Comment:
NOTE, this feature will be leaked into `dependencies` (it's fine but still.. the new feature resolver[https://github.com/rust-lang/cargo/pull/8997) is stabilized and will fix this FWIW)